jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,289 @@ version:
 - Cloud
 ---
 
+* TOC
+{:toc}
+
 ## Introduction
 
-Once you have configured your platform to work with CircleCI orbs, you are ready to begin authoring your own orb. CircleCI has attempted to make this process simple and easy, eliminating any potential confusion by enabling you to use inline templates and other pre-configured templates to quickly author a new orb. The sections below describe the process of authoring your own orb.
+  Once you have read the [Introduction to authoring an orb]({{site.baseurl}}/2.0/orb-intro) document and claimed your namespace, you are ready to develop and orb.
 
-## Orbs Quickstart
+  As orbs are simply packages of [reusable config]({{site.baseurl}}/2.0/reusing-config), orbs can be written [manually]({{site.baseurl}}/2.0/orb-author-validate-publish) as singular `yaml` files and published using our [circleci orb cli]({{site.baseurl}}/2.0/local-cli/#installation).
 
-Before you create your first orb, please see the following notes:
+  Alternatively, whether you are writing your first orb or getting ready for production level, we recommend using our Orb Development Kit to get started.
 
-* Orbs exist in namespaces.
-* Each organization or username can claim one unique namespace, and **namespaces cannot be deleted.**
-* Namespaces are global within the CircleCI orb registry, so pick a unique name!
-* Only users with "owner"/"administrator" privileges within a given GitHub/Bitbucket organization can create a namespace linked to that organization.
-* Once an orb has been created by an org admin, any org member can publish `dev` orbs. When ready, `dev` orbs can be promoted into production by org admins.
-* When stored as a project environment variable or Contexts resource, an [API token]({{ site.baseurl }}/2.0/managing-api-tokens) generated by an org admin can help automate the orb publishing process, enabling _any_ org member to potentially publish a production orb by using the [CircleCI CLI]({{ site.baseurl }}/2.0/local-cli/) in a CircleCI job.
-* To use the orbs you create, you will need to enable the "Allow Uncertified Orbs" setting in the Security section of CircleCI's Organization Settings page for your organization (`https://circleci.com/[vcs]/organizations/[org-name]/settings#security`).
-* If you are creating an orb, you can use development versions to avoid having your orb publicly/permanently listed in CircleCI's orb registry before it is ready.
+## Orb Development Kit
 
-### The following high-level steps are needed to publish your first orb:
+The Orb Development Kit refers to a suite of tools working together to make production-level orb development simple, with automatic testing and deployment on CircleCI.
 
-1) Claim a namespace (assuming you don't yet have one). For example:
+### Getting Started
 
-`circleci namespace create sandbox github CircleCI-Public`
+To begin creating your new orb with the Orb Developer Kit, first create a new repository on [GitHub.com](https://github.com).
```

Comment:
Does it have to be github? not bitbucket?

Reply:
There is actually a good chance it works on BitBucket now (we removed a lot of code that was GitHub dependant) however... we have not _yet_ tested on BitBucket.

Reply:
👍